src/debug/inc/arm/primitives.h

```diff
@@ -30,7 +30,11 @@ typedef DPTR(CORDB_ADDRESS_TYPE)    PTR_CORDB_ADDRESS_TYPE;
 #define STACKWALK_CONTROLPC_ADJUST_OFFSET 2
 
 #define CORDbg_BREAK_INSTRUCTION_SIZE 2
+#ifdef LINUX32
```

Comment:
Ok, let's make it that way for now. We don't support non-Linux ARM32 Unix platforms anyways, so once someone comes and will try to add support for such a platform, it can be revised.